userguide/services.rst

```diff
@@ -757,9 +757,69 @@ Netdata
+   +--------------------------+----------------+----------+-------------------------------------------------------------------------------+
+   | Additional Parameters    | string         | ✓        | Define other sections and their key/value pairs. Enclose each section name in |
+   |                          |                |          | square brackets and put each key/value pair on a new line. Example:           |
+   |                          |                |          |                                                                               |
+   |                          |                |          | .. code-block:: none                                                          |
+   |                          |                |          |                                                                               |
+   |                          |                |          |    [system.intr]                                                              |
+   |                          |                |          |    history=86400                                                              |
+   |                          |                |          |    enabled=yes                                                                |
+   |                          |                |          |                                                                               |
+   +--------------------------+----------------+----------+-------------------------------------------------------------------------------+
+   | Alarms                   | drop-down menu | ✓        | Click on alarms to select or unselect.                                        |
+   +--------------------------+----------------+----------+-------------------------------------------------------------------------------+
+   | Stream Mode              | drop-down menu | ✓        | Select a stream mode if the system is to be used for streaming.               |
+   +--------------------------+----------------+----------+-------------------------------------------------------------------------------+
+   | Destination              | string         | ✓        | Only appears when *Slave* is the :guilabel:`Stream Mode`. Please provide a    |
+   |                          |                |          | line- or space-separated list of destinations where the collected metrics are |
+   |                          |                |          | to be sent. Use the format HOST:PORT (port is optional. Netdata uses the      |
+   |                          |                |          | first working destination.                                                    |
+   +--------------------------+----------------+----------+-------------------------------------------------------------------------------+
+   | API Key                  | string         | ✓        | Only appears when *Slave* or *Master* is the :guilabel:`Stream Mode`. The     |
```

Comment:
Put description first: "API_KEY to use as the sender..."  Move the current first sentence to the end, say "This field only appears..."